core/src/com/cloud/agent/api/SecurityGroupRulesCmd.java

```diff
@@ -35,7 +35,7 @@
 public class SecurityGroupRulesCmd extends Command {
     private static final String CIDR_LENGTH_SEPARATOR = "/";
     private static final char RULE_TARGET_SEPARATOR = ',';
-    private static final char RULE_COMMAND_SEPARATOR = ':';
+    private static final char RULE_COMMAND_SEPARATOR = ';';
```

Comment:
@wido can changing the separator cause issues (upgrade issues) for existing environments?